packages/alert-banner/README.md

```diff
@@ -1,64 +1,71 @@
-## Description
+## Overview
 
-The `<sp-alert-banner>` shows pressing and high-signal messages, such as system alerts. It is meant to be noticed and prompt users to take action.
+The `<sp-alert-banner>` displays pressing and high-signal messages, such as system alerts. It is intended to be noticeable and prompt users to take action.
 
 ### Usage
 
 [![See it on NPM!](https://img.shields.io/npm/v/@spectrum-web-components/alert-banner?style=for-the-badge)](https://www.npmjs.com/package/@spectrum-web-components/alert-banner)
 [![How big is this package in your project?](https://img.shields.io/bundlephobia/minzip/@spectrum-web-components/alert-banner?style=for-the-badge)](https://bundlephobia.com/result?p=@spectrum-web-components/alert-banner)
 
-```
+```bash
 yarn add @spectrum-web-components/alert-banner
 ```
 
 Import the side effectful registration of `<sp-alert-banner>` via:
 
-```
+```javascript
 import '@spectrum-web-components/alert-banner/sp-alert-banner.js';
 ```
 
-When looking to leverage the `AlertBanner` base class as a type and/or for extension purposes, do so via:
+### Anatomy
 
-```
-import { AlertBanner } from '@spectrum-web-components/alert-banner';
-```
+The alert dialog consists of several key parts:
 
-## Examples
+#### Content
 
-The alert banner accepts text context in the default slot provided:
+The message in its default slot.
 
 ```html
 <sp-alert-banner open>
     All documents in this folder have been archived
 </sp-alert-banner>
 ```
 
-### Dismissible
+#### Action Buttons
```

Comment:
i think this should move to options based on the design docs: https://spectrum.adobe.com/page/alert-banner/

or dismissable moves up to anatomy

Reply:
design docs are confusing in their language so this can be a `nit`, this technically aligns with the usage guidelines in design docs. confusion comes from the options section in design docs.